content/applications/sales/crm/pipeline/manage_sales_teams.rst

```diff
@@ -0,0 +1,106 @@
+==================
+Manage sales teams
+==================
+
+The *Sales Teams* feature within Odoo's *CRM* app allows for the creation and management of multiple
+sales teams, each with their own assignment rules, invoicing targets, and roster of salespeople.
+
+Create a sales team
+===================
+
+To create a new sales team, go to :menuselection:`CRM app --> Configuration --> Sales Teams`, then
+click :guilabel:`New`.
+
+On the blank sales team form, enter a name in the :guilabel:`Sales Team` field.
+
+Next, select a :guilabel:`Team Leader` from the drop-down list.
+
+Set an :guilabel:`Email Alias` to automatically generate a lead/opportunity for this sales team
+whenever a message is sent to that unique email address. Choose whether to accept emails from
+:guilabel:`Everyone`, :guilabel:`Authenticated Partners`, :guilabel:`Followers Only`, or
+:guilabel:`Authenticated Employees`.
+
+Select a :guilabel:`Company` from the drop-down to assign this team to. *This field is only visible
+in multi-company databases, and is not required.*
+
+.. image:: manage_sales_teams/sales-team-creation.png
+   :align: center
+   :alt: The settings page for a new sales team.
+
+.. note::
+   If the *Sales* app is installed on the database, an :guilabel:`Invoicing Target` field appears on
+   the sales team form. This is the revenue target for the current month. The amount entered in this
+   field is used to populate the invoicing progress bar on the :ref:`sales team dashboard
+   <crm/sales-team-dashboard>`.
+
+Add sales team members
+----------------------
+
+To add team members, click :guilabel:`Add` under the :guilabel:`Members` tab when editing the sales
+team's configuration page. This opens a :guilabel:`Create Sales Team Members` pop-up window.
+
+.. note::
+   If the :guilabel:`Rule-Based Assignment` feature has **not** been enabled on the *CRM* app's
+   *Settings* page, clicking :guilabel:`Add` under the :guilabel:`Members` tab opens an
+   :guilabel:`Add\:\  Salespersons` pop-up window. Tick the checkbox to the far-left of the
+   salesperson to be added to the team, then click :guilabel:`Select`.
+
+   .. image:: manage_sales_teams/add-salespersons.png
```

Comment:
I think this image would be more useful if a salesperson was selected, like the instructions detail, this would also allow the **Select** button to not be grayed out
```suggestion
   .. image:: manage_sales_teams/add-salespersons.png
```